src/mono/mono/component/debugger-agent.c

```diff
@@ -4302,8 +4301,8 @@ mono_de_frame_async_id (DbgEngineStackFrame *frame)
 	return get_objid (obj);
 }
 
-static gboolean
-begin_breakpoint_processing (void *the_tls, MonoContext *ctx, MonoJitInfo *ji, gboolean from_signal)
+bool
```

Comment:
Why dropping static here and moving the function out of the file?

Reply:
Because I'm calling it from mini-wasm-debugger :)

Reply:
"Static functions in C are functions that are restricted to the same file in which they are defined". Makes sense.